demosplan/DemosPlanCoreBundle/Command/Addon/AddonInstallFromZipCommand.php

```diff
@@ -503,4 +517,32 @@ function ($tag) {
 
         return $this->getHelper('question')->ask($input, $output, $question);
     }
+
+    private function getPathFromLocalDevelopment(InputInterface $input, SymfonyStyle $output, mixed $folder): string
+    {
+        $fs = new Filesystem();
+        $addonDevFolder = DemosPlanPath::getRootPath('addonDev');
+        if (!file_exists($addonDevFolder)) {
+            throw new RuntimeException("No folder {$folder} found. To develop addons locally, create a folder {$folder} and put your addons in there.");
+        }
+
+        $localAddons = glob($addonDevFolder.'/*');
+        if (!is_array($localAddons) || 0 === count($localAddons)) {
+            throw new RuntimeException("No local addons found in folder {$folder}. Please check out the demosplan-addon-* repositories into this folder.");
```

Comment:
The same happens after I created a empty addonDev folder. Have you an Idea why the addonZips folder name is used?